src/components/views/messages/TextualBody.js

```diff
@@ -37,28 +38,76 @@ module.exports = React.createClass({
 
         /* link URL for the highlights */
         highlightLink: React.PropTypes.string,
+
+        /* callback for when our widget has loaded */
+        onWidgetLoad: React.PropTypes.func,
+    },
+
+    getInitialState: function() {
+        return {
+            link: null,
```

Comment:
The only thing that might be nice would be a comment on `link` in `getInitialState` saying what it does.